man/rdtd.conf.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version='1.0'?>
```

Comment:
After adding any man pages, run `ninja -C build man/update-man-rules` to update the autogenerated meson.build files.